deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -100,7 +100,7 @@ class Agent:
     """
     def __init__(self, manager_address, cypher="aes", os=None, inventory_sample=None, rootcheck_sample=None,
                  id=None, name=None, key=None, version="v3.12.0", labels=None, fim_eps=None, fim_integrity_eps=None,
-                 authd_password=None, disable_all_modules=False, rcv_msg_limit=999):
+                 authd_password=None, disable_all_modules=False, rcv_msg_limit=0):
```

Comment:
Have you taken this change into account for the other uses of this queue, e.g. remoted tests?

Reply:
Yes, 0 means no limit, this prevents various tests from failing due to a full queue stoping the receive_events process. (When the queue is full, the process is blocked until another process removes elements from it. If there are no other processes reading from the queue, the process is hanged in an infinite loop, so it is better to have an unlimited queue).